r-package/grf/_pkgdown.yml

```diff
@@ -0,0 +1,76 @@
+url: https://github.com/grf-labs/grf
+navbar:
+  type: default
+  right:
+  - icon: fa-github fa-lg
+    href: https://github.com/grf-labs/grf
+
+## Function reference
+# Different forest types
+reference:
+  - title: Causal forest
```

Comment:
I think it'd be nice to have `Regression forest` first, then `Quantile forest` second, and everything else in the same order below.